easybuild/easyconfigs/m/Mash/Mash-2.3-GCC-12.3.0.eb

```diff
@@ -0,0 +1,48 @@
+# Contribution by
+# DeepThought, Flinders University
+# Updated to v2.2
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'ConfigureMake'
+
+name = 'Mash'
+version = '2.3'
+
+homepage = 'http://mash.readthedocs.org'
+description = "Fast genome and metagenome distance estimation using MinHash"
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+toolchainopts = {'cstd': 'c++17'}
```

Comment:
```suggestion
```

From https://gcc.gnu.org/projects/cxx-status.html#cxx17
> C++17 mode is the default since GCC 11

I've opened #18941 to remove this from the GCC 11.x Mash easyconfigs.